Reconstruction/RecCalorimeter/tests/options/runFullCaloSystem_SimAndDigitisation.py

```diff
@@ -208,26 +197,7 @@
                                addCellNoise = False, filterCellNoise = False,
                                OutputLevel = INFO,
                                hits="HCalHits",
-                               cells="HCalCells")
-
-# additionally for HCal
-positionsHcal = CreateVolumeCaloPositions("positionsHcal", OutputLevel = INFO)
-positionsHcal.hits.Path = "HCalCells"
-positionsHcal.positionedHits.Path = "HCalPositions"
-
-from Configurables import RedoSegmentation
```

Comment:
This should remain, otherwise hcal cells will not be appended with eta-phi segmentation and will be useless for sliding window reconstruction

Reply:
Hi Anna, thanks for the comments! 
For the re-segmentation, I have added it to the SW reco.